File: src/main/java/no/fintlabs/linkwalker/auth/model/TokenResponse.java
package no.fintlabs.linkwalker.auth.model;

import com.fasterxml.jackson.annotation.JsonProperty;

public record TokenResponse (
        @JsonProperty("access_token") String accesToken,
        @JsonProperty("token_type") String tokenType,
        @JsonProperty("expires_in") int expiresIn,
        String acr,
        String scope
) {
}

File: src/main/kotlin/no/fintlabs/linkwalker/auth/model/TokenResponse.kt
package no.fintlabs.linkwalker.auth.model

import com.fasterxml.jackson.annotation.JsonProperty

data class TokenResponse(
    @JsonProperty("access_token")
    val accessToken: String,
    @JsonProperty("token_type")
    val tokenType: String,
    @JsonProperty("expires_in")
    val expiresIn: Int,
    val acr: String,
    val scope: String
)
